packages/dynamite/dynamite/lib/src/builder/client.dart

```diff
@@ -554,7 +553,7 @@ String buildParameterSerialization(
   String Function(Reference) allocate,
 ) {
   final dartName = toDartName(parameter.name);
-  final serializedName = '\$$dartName';
+  final serializedName = '__$dartName';
```

Comment:
To avoid name clashes when is a request body which will already be called `$body`. A single underscore won't work either, so double it is. I wonder if there is some magic that we could use to avoid this stuff and simply allocate variables without having to think about clashing names.